### ISSUE TITLE
The text "Home timeline for ..." should be a true heading for easier screen reader navigation.

### ISSUE THREAD

**Issue by MarcoZehe** [nolanlawson/pinafore#44]
The text that precedes the actual posts of a timeline should be a heading. That way, screen reader users can jump past the navigation and toot composer straight to the beginning of their timeline and start reading. Right now, this is plain text.

**Comment by nolanlawson** [comment]
Thanks for the issue! This was a dumb mistake on my part; should be a quick fix. 😊
